pkg/apis/v1alpha1/ecsnodeclass.go

```diff
@@ -250,7 +250,7 @@ type ECSNodeClass struct {
 // 1. A field changes its default value for an existing field that is already hashed
 // 2. A field is added to the hash calculation with an already-set value
 // 3. A field is removed from the hash calculations
-const ECSNodeClassHashVersion = "v1"
+const ECSNodeClassHashVersion = "v3"
 
```

Comment:
Why change this?

Reply:
Maybe we don't need to change it. I'll test it again.